test/semantics/CMakeLists.txt

```diff
@@ -152,6 +152,10 @@ set(IF_TESTS
   if_stmt01.f90
 )
 
+set(ANY_TESTS
```

Comment:
I don't think we're ready for another generic category (ANY == generic).  You can add your test under ERROR_TESTS; it's fine if it doesn't get an error.